README.md

```diff
@@ -1,2 +1,2 @@
 # simple-node-server
-Create simple node server
+Create simple node server [![Build Status](https://travis-ci.org/kavitamakariye/simple-node-server.svg?branch=master)](https://travis-ci.org/kavitamakariye/simple-node-server)
```

Comment:
Markdown URL should be [![Build Status](https://travis-ci.org/Hdonge/simple-node-server.svg?branch=master)](https://travis-ci.org/Hdonge/simple-node-server) of main upstream repo

Reply:
![image](https://user-images.githubusercontent.com/17523943/59150803-e2dd2b80-8a46-11e9-99db-9b7d6ee9a83b.png)

Reply:
Changed the Markdown to upstream repo